src/main/java/mops/persistence/file/FileInfo.java

```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
     }
 
     /**
-     * Gives you FileInfoBuilder.
+     * Converts Byte to a String.
      *
-     * @return FileInfoBuilder
+     * @return Filesize with prefix.
      */
-    public static FileInfoBuilder builder() {
-        return new FileInfoBuilder();
+    public String getSizeString() {
+        final double kilobyte = 1024;
+        final double megabyte = kilobyte * 1024;
+        final double gigabyte = megabyte * 1024;
+
+        double result;
+        String suffix;
+
+        if (this.size < kilobyte) {
+            result = this.size;
+            suffix = " B";
+        } else if (this.size > kilobyte && this.size < megabyte) {
+            result = this.size / kilobyte;
+            suffix = " Kb";
```

Comment:
```suggestion
            suffix = " KB";
```
Kb = Kilobit
KB = Kilobyte